src/main/kotlin/blackJack/domain/DealerResult.kt

```diff
@@ -0,0 +1,21 @@
+package blackJack.domain
+
+enum class DealerResult(private var count: Int) {
```

Comment:
다른 개발자도 예상할 수 있는 기능일까요? 아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/kotlin-blackjack/pull/21#discussion_r469610136

```suggestion
enum class DealerResult {
```